src/dev/konnector.md

```diff
@@ -39,179 +56,168 @@ yarn # or npm install
 
 _note: the Cozy Team uses `yarn`, but if you prefer `npm`, just keep using it, everything should just work._
 
-Then, write your code into `index.js` and build the application, running `yarn build` or `npm run build`.
+The connector template already comes with some functionnal and commented code which scrapes
```

Comment:
s/functionnal/functional/

Reply:
d/some/ . Some is very vague. It does not look good in a doc :)

Reply:
The connector template is ready with demo code to show you how to scrape a fictional
website.